web/static/ivre/graph.js

```diff
@@ -94,6 +94,10 @@ var GraphTopValues = (function(_super) {
 	this.filename = "TopValues";
     }
 
+    function escapeRegex(string) {
+        return '\\"'+string.replace(/[-\/\\^$*+?.()|[\]{}]/g, '\\\\$&').trim()+'\\"';
+    }
```

Comment:
I'd rather remove the double quotes added here as they are not part of the regexp escape process:
```suggestion
// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#escaping
function escapeRegExp(string) {
  return string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
}
```

Reply:
The double quote is here to take the string in it's whole, and simplify the condition and make more readable, and also to prevent repeat the double quote everytime.
Maybe we should name it formatRegEx instead of escapeRegEx, and set a parameter to include the double quote or not ?
But you remove trim, and it's necessary to remove the line feed, if not the Javascript will fail and drop an error "unescaped line feed"

Reply:
Good point for the `.trim()`. I think it is better to add it after the call, so I did it (see below)